scoper.inc.php

```diff
@@ -142,6 +142,19 @@
 			->in( 'vendor/sabberworm/php-css-parser' )
 			->append( [ 'vendor/sabberworm/php-css-parser/composer.json' ] ),
 
+		Finder::create()
+			->files()
+			->ignoreVCS( true )
+			->ignoreDotFiles( true )
+			->name( '*.php' )
+			->exclude(
+				[
+					'tests',
+				]
+			)
+			->in( 'vendor/enshrined/svg-sanitize' )
```

Comment:
```suggestion
			->in( 'vendor/enshrined/svg-sanitize/src' )
```

Otherwise this includes `svg-scanner.php`, a small PHP command line utility that could be potentially dangerous to include in the plugin.